generated/nifgen/session.py

```diff
@@ -1969,6 +1969,31 @@ def delete_script(self, script_name):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def delete_waveform(self, waveform_name_or_handle):
+        '''delete_waveform
+
+        Removes a previously created arbitrary waveform from the signal generator memory.
+
+        Note: The signal generator must not be in the Generating state when you call this method.
+
+        Tip:
+        This method requires repeated capabilities (usually channels). If called directly on the
```

Comment:
A lot of methods in NI-FGEN have channel inputs that are there for future proofing or are used +90% of the time with empty string.

Documentation bolerplate says "requires". This makes me uneasy.

This is food for thought outside of this PR.